chart/README.md

```diff
@@ -191,3 +191,55 @@ $ helm install my-release openebs/mayastor
 | priorityClassName | Pod scheduling priority. Setting this value will apply to all components except the external Chart dependencies. If any component has `priorityClassName` set, then this value would be overridden for that component. For external components like etcd, jaeger or loki-stack, PriorityClass can only be set at component level. | `""` |
 | tolerations | Tolerations to be applied to all components except external Chart dependencies. If any component has tolerations set, then it would override this value. For external components like etcd, jaeger and loki-stack, tolerations can only be set at component level. | `[]` |
 
+# nats
```

Comment:
Why is this being included here?

Reply:
This whole thing is being added automatically by the pre-commit.

Reply:
The script is picking up on the chart files of the nats subchart. The subchart was likely pulled in with a `dependency update`, and unpacked in-place.

Removing the subchart dir should generate the README.md correctly, I think.

Reply:
Yes, removing the chart/charts directory works.